tests/integration/kubernetes/setup.sh

```diff
@@ -23,6 +23,13 @@ set_hypervisor_path() {
     fi
 }
 
+set_kernel_path() {
+    if [[ "${KATA_HOST_OS}" = "cbl-mariner" ]]; then
+        mariner_kernel_path="/boot/vmlinux.bin"
+        find runtimeclass_workloads/*.yaml -exec yq write -i {} 'metadata.annotations[io.katacontainers.config.hypervisor.kernel]' "${mariner_kernel_path}" \;
```

Comment:
FIXME: This won't have an effect because `kernel` isn't in `enable_annotations`.